css/style.css

```diff
@@ -16,4 +16,257 @@ body {
+    padding-top: 0.5rem;
+}
+
+.icon:hover {
+    color:indianred;
+    transition: all 0.3s ease;
+}
+
+.icon.cancel-btn {
+    position: absolute;
+    right: 3rem;
+    top: 2rem;
+    transition: all 0.3s ease;
+}
+
+@media (max-width: 768px) {
+
+    body.disabledScroll {
+        overflow: hidden;
+    }
+
+    .navbar .menu-list {
+        position: fixed;
+        height: 100vh;
+        width: 100%;
+        max-width: 400px;
+        left: -100%;
+        top: 0;
+        background: #222;
+        display: block;
+        padding: 1rem 0;
+        text-align: center;
+        transition: all 0.3 ease;
+    }
+
+    .navbar {
+        display: flex;
+        align-content: center;
+        justify-content: space-between;
+    }
+
+    .navbar .menu-list.active {
+        left: 0%;
+    }
+
+    .navbar .menu-list li{
+        margin-top: 2rem;
+        
+    }
+
+    .navbar .menu-list a {
+        color: rgb(190, 110, 35);
+    }
+
+    .menu-list li a:hover {
+        color: antiquewhite;
+    }
+
+    .icon {
+        display: block;
+    }
+
+    .icon.hide {
+        display: none;
+    }
+
+    .menu-btn {
+        left: 0%;
+    }
+
+    .icon.cancel-btn {
+        color: rgb(190, 110, 35);
+    }
+
+    .icon.cancel-btn:hover {
+        color: antiquewhite;
+    }
+
+    .container-devices {
+        flex-direction: column;
+    }
+
+}
+  
+
+.hero {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background: url("../img/first-background.jpg") no-repeat;
+    background-size: cover;
+    height: 100vh;
+    background-position: center;
+    background-attachment: fixed;
+ }
+
+.hero-content {
+    color: honeydew;
+    line-height: 3rem;
+    /* font-size: 1.3rem; */
+}
+
+.hero-content h1 {
+    font-size: 3rem;
+    font-weight: 300;
+    text-align: center;
+}
+
+.hero-content p {
+    font-size: 2rem;
+    font-weight: 100;
+    text-align: center;
+}
+
+button {
+    color: #fff;
+    font-weight: 400;
+    background-color: rgb(189, 73, 27);
+    border-radius: 10px;
+    font-size: larger;
+    margin-top: 2rem;
+    width: 9rem;
+    border-style: none;  
+    position: relative;
+    left: 33%    
+}
+
+.section-devices {
+    margin-bottom: 5rem;
+}
+
+.section-devices h1 {
+    margin-top: 4rem;
+    text-align: center;
+    font-weight: 400;
+} 
+
+.container-devices {
+    display: flex;
+    justify-content: space-around;
+    align-content: center;
+    text-align: center;
+    margin: auto 15%;
+    margin-top: 2rem;
+    flex-wrap: wrap;
+}
+
+.devices {
+    width: 25%;
+    height: 20vh;
+    margin: 1rem;
+}
+
+.devices img {
+    width: 50%;
+}
+
+footer {
+    margin-top: 3rem;
+    border-top:2px solid rgb(201, 98, 38);
+}
+
+footer h6 {
+    margin-top: 1rem;
+    text-align: center;
+    font-size: 1rem;
+}
+
+.social-media-images {
+    /* display: flex; */
+    flex-direction: row;
+    margin-top: 1rem;
+}
+
+.social-media-images img {
+    width: 7%;
```

Comment:
You might want to have more control over the size of your icons. It's common to use fixed sizes for icons, such as pixels (`px`) or rems (`rem`), rather than percentages.

Reply:
Fixed.

.social-media-images img {
    width: 2rem;